packages/typescript-estree/src/parser-options.ts

```diff
@@ -1,11 +1,25 @@
 import { Program } from 'typescript';
 import { TSESTree, TSNode } from './ts-estree';
 
+type DebugModule = 'typescript-eslint' | 'eslint' | 'typescript';
+/**
+ * For convenience:
+ * - true === ['typescript-eslint']
+ * - false === []
+ *
+ * An array of modules to turn explicit debugging on for.
+ * - 'typescript-eslint' is the same as setting the env var `DEBUG=typescript-eslint:*`
+ * - 'eslint' is the same as setting the env var `DEBUG=eslint:*`
+ * - 'typescript' is the same as setting `extendedDiagnostics: true` in your tsconfig compilerOptions
```

Comment:
can you add this note somewhere in docs?

Reply:
I'm working on updating the ts-estree docs, as they're rather outdated.
I'll submit the readme update as a separate PR.